coil-compose-core/src/commonMain/kotlin/coil3/compose/ConstraintsSizeResolver.kt

```diff
@@ -29,21 +29,23 @@ fun rememberConstraintsSizeResolver(): ConstraintsSizeResolver {
  */
 @Stable
 class ConstraintsSizeResolver : SizeResolver, LayoutModifier {
-    private val latestConstraints = MutableSharedFlow<Constraints>(
-        replay = 1,
-        onBufferOverflow = DROP_OLDEST,
-    )
+    private var latestConstraints: Constraints = Constraints(maxWidth = 0, maxHeight = 0)
```

Comment:
Awesome! TIL you can get some extra performance by manually resuming the `Continuation`.